schema/windows-view-model.json

```diff
@@ -25,6 +25,9 @@
         "certificates": {
             "type": "array"
         },
-        "cookiePromptManagementStatus": { "$ref": "./cookie-prompt-management-status.json" }
+        "cookiePromptManagementStatus": { "$ref": "./cookie-prompt-management-status.json" },
+        "isInvalidCert": {
+            "type": "boolean"
+        }
```

Comment:
My only feedback in the entire PR, is that having this as a boolean might be too restrictive in the future (for example, if we want different errors for different kinds of invalid certs). 

However, because this would be VERY easy to upgrade to an enum later (if we actually need it), I don't think you should alter it now :)

Likewise I'd prefer it if all certificate information were under a single structure, instead of separate - but the legacy support of all the other platforms makes that difficult. Again, nothing to alter in your PR, just highlighting where I think improvements can come later.